opportunity.py

```diff
@@ -1,23 +1,10 @@
 from dataclasses import dataclass
 from dotenv import load_dotenv
-import gpt4free
-from gpt4free import Provider
-from typing import List, Optional
-import os
-import ast
-from time import sleep
-import json
+from typing import List
 import utility as utils
-import openai
 
 load_dotenv()
 
-openai.api_key = os.getenv("GPT_API_KEY")
-
-table_name = os.getenv("DB_TABLE")
-
-MAX_RETRY = 15  # Max number of retrys for the gpt_job_analyzer() function
-
 
 @dataclass
 class Opportunity:
```

Comment:
**TO-DO**
- Change all the instances of the Opportunity class to without the underscore prefix
- Add a column to the `jobs_table` table with the column name of `type`
- Combine the two tables into one large table for all types of opportunities
- Every instance that the object is being constructed, do not forget to set the type of opportunity
@boushrabettir

Reply:
Implemented [here](https://github.com/acmcsufoss/acmOpportunities/pull/11/commits/a0efeafe1f0d4b3cdcfac384dccd23bfeb895bba).